shared-functions

```diff
@@ -435,16 +435,15 @@ upload_stemcell() {
   local stemcell_name
 
   local infrastructure
-  if [ -d toolsmiths-env ]; then
+  if [ "${BOSH_LITE}" == "true"  ]; then
+    infrastructure="bosh-lite"
+    stemcell_name="bosh-warden-boshlite"
+  elif [ -d toolsmiths-env ]; then
     infrastructure="gcp"
     stemcell_name="bosh-google-kvm"
   else
     infrastructure="$(jq -r .iaas bbl-state/${BBL_STATE_DIR}/bbl-state.json)"
 
-    if [ "${BOSH_LITE}" == "true"  ]; then
-      infrastructure="bosh-lite"
-    fi
-
     if [ "$infrastructure" = "bosh-lite" ]; then
       stemcell_name="bosh-warden-boshlite"
```

Comment:
Since you are setting stemcell_name above, I think this check can be removed as well, right?

Reply:
I thought there might be a case where `bosh-lite` could be detected from the `bbl-state`. Was that an incorrect assumption?

Reply:
I do see that snitch [uses](https://github.com/cloudfoundry/cf-deployment/blob/04da087800ce57b123c92e4862b3161219af67ce/ci/pipelines/cf-deployment.yml#L1415) the param... okay I'll remove this check.